pkg/controllers/csi/driver/volumes/app/publisher_test.go

```diff
@@ -313,7 +314,7 @@ func TestMountIfDBHasError(t *testing.T) {
 }
 
 func newPublisherForTesting(mounter *mount.FakeMounter) AppVolumePublisher {
-	csiOptions := dtcsi.CSIOptions{RootDir: "/"}
+	csiOptions := dtcsi.CSIOptions{RootDir: "/data/"}
```

Comment:
I don't understand why do we want this prefix?

Reply:
Not 100% about it but I think was a false positive from previous implementation. I wrote more about it [here](https://github.com/Dynatrace/dynatrace-operator/pull/2906#discussion_r1569176452).

Reply:
As tested yesterday, this is the issue of why the E2E test is failing. Mount paths are a bit flaky and I'm having a look at it at the moment :+1:

Reply:
This change was reverted :+1: closing the discussion